loadflow/loadflow-api/src/main/java/com/powsybl/loadflow/LoadFlowResultImpl.java

```diff
@@ -28,12 +30,13 @@
 
         private final double slackBusActivePowerMismatch;
 
-        public ComponentResultImpl(int componentNum, Status status, int iterationCount, String slackBusId, double slackBusActivePowerMismatch) {
-            this.componentNum = checkComponentNum(componentNum);
+        public ComponentResultImpl(int connectedComponentNum, int synchronousComponentNum, Status status, int iterationCount, String slackBusId, double slackBusActivePowerMismatch) {
```

Comment:
@flo-dup I am not always familiar with `breaking change` but I can imagine that it is one. So it means that we need to deal with decrease version compatibility?

Reply:
Yes probably. Don't know the process here, so waiting for @flo-dup or @geofjamg insight.

Reply:
You have increase the json version (1.2) and adapt the deserializer to support all versions. We can consider than component number in the actual code is synchronous component number in the new one.

Reply:
we have to deal with version compatibility, but no need no deprecate constructor.